lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -400,7 +489,7 @@ def header_mappings_dir
                                    end
           end
 
-          def add_header(build_file, public_headers, private_headers)
+          def add_header(build_file, public_headers, private_headers, native_target)
```

Comment:
It is, the method uses `native_target` which is a property but now the parameter takes over so no real method changes were required.